task-3_mongoDB-p1/README.md

```diff
@@ -0,0 +1,92 @@
+```
+mongoimport --db frontcamp --collection restaurants --file C:/data/db/restaurants.json
+```
```

Comment:
Please make sure that collection is correctly imported according to 2.4

Reply:
What is wrong with import?

Reply:
Please attach a screenshot with number of the documents in the collection equals 25359

Reply:
![image](https://user-images.githubusercontent.com/45118964/68540764-3eba3080-03a8-11ea-844b-8d193682d02f.png)